### DISCUSSION TITLE
ezcox画图问题

### DISCUSSION THREAD

**Discussion by zhaoliang0302** [ShixiangWang/self-study#46] [Q&A] [answered]
翔哥好，我在使用你的ezcox包时出图发现黑色方块太大，请问可以把方块调小吗？
还有可以不显示N总数这一栏吗？
谢谢

**Comment by zhaoliang0302** [comment]
![image](https://user-images.githubusercontent.com/42333702/80791393-188d8480-8bc4-11ea-9574-9a1cf9a7a070.png)

**Comment by zhaoliang0302** [comment]
方块大小和HR相关，我知道了😂
那这个N怎么去掉呢？

**Comment by ShixiangWang** [comment]
可以进行设置 我有点忘了 函数里面有一个参数可以传入自定义的 可能有点难搞喔 为啥不显示样本数呢

**Comment by ShixiangWang** [comment]
底层实现是基于 forestmodel 很多东西可能需要改底层依赖包 有空我看下怎么调小吧

**Comment by zhaoliang0302** [comment]
嗯嗯，我可以AI调整下，谢谢你开发好用的r包

**Comment by ShixiangWang** [comment, marked as answer]
@zhaoliang0302 https://github.com/ShixiangWang/ezcox/issues/12 查看提供的实例去修改方块大小

**Comment by zhaoliang0302** [comment]
非常感谢！
